ExposureNotification.App/ExposureNotification.App/Services/ExposureNotificationHandler.cs

```diff
@@ -20,16 +20,17 @@ public class ExposureNotificationHandler : IExposureNotificationHandler
 
 		static readonly HttpClient http = new HttpClient();
 
+		public string UserExplanation
+			=> "We need to make use of the keys to keep you healthy.";
+
 		public Task<Configuration> GetConfigurationAsync()
 			=> Task.FromResult(new Configuration());
 
-		public async Task ExposureDetected(ExposureDetectionSummary summary, Func<Task<IEnumerable<ExposureInfo>>> getDetailsFunc)
+		public async Task ExposureDetectedAsync(ExposureDetectionSummary summary, IEnumerable<ExposureInfo> exposureInfo)
```

Comment:
Technically we could go back to a delegate here... I changed it during the first pass, but left it...